### PR TITLE
Update nodejs version to 12 - Closes #356 

### LINKED ISSUE #356: Update NodeJS version to 12 in documentation
We updated Node to version 12 via https://github.com/LiskHQ/lisk-sdk/pull/4448 we need to update the documentation to reflect this change.

### PR DESCRIPTION
**Pull request by chris529**
Closes #356

### CODE REVIEW

File: modules/ROOT/pages/setup/source.adoc [hunk 1]
```diff
@@ -94,7 +94,7 @@ brew install git
 === Node.js
 
 {url_nodejs}[Node.js] serves as the underlying engine for code execution.
-There are several different methods and version managers used to install Node.JS on your system.
+There are several different methods and version managers used to install Node.JS on your system. Please ensure the latest version is used.
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/source.adoc]: Add `LTS` here.

```suggestion
There are several different methods and version managers used to install Node.JS on your system. Please ensure the latest LTS version is used.
```

File: modules/ROOT/pages/setup/source.adoc [hunk 2]
```diff
@@ -106,17 +106,17 @@ It is recommended to use a Node version manager such as {url_nvm}[NVM].
 NVM is a bash script that enables the management of multiple active Node.js versions.
 
 . Install nvm following the {url_nvm_instructions}[official instructions]
-. Install the correct version of Node.js using NVM:
+. Install the latest version of Node.js using NVM:
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/source.adoc]:
```suggestion
. Install the latest LTS version of Node.js using NVM:
```

**Review by Tschakki — CHANGES_REQUESTED** [review]

**Review by Tschakki — APPROVED** [review]
Thanks!

File: modules/ROOT/pages/setup/source.adoc [hunk 3]
```diff
@@ -106,17 +106,17 @@ It is recommended to use a Node version manager such as {url_nvm}[NVM].
 NVM is a bash script that enables the management of multiple active Node.js versions.
 
 . Install nvm following the {url_nvm_instructions}[official instructions]
-. Install the correct version of Node.js using NVM:
+. Install the latest LTS version of Node.js using NVM:
 
 [source,bash]
 ----
-nvm install 10.15.3
+nvm install 12.14.1
```
Comment (chris529) [inline modules/ROOT/pages/setup/source.adoc:113]: 12.15.1

File: modules/ROOT/pages/setup/source.adoc [hunk 4]
```diff
@@ -130,14 +130,14 @@ sudo apt-get install -y nodejs
 
 [source,bash]
 ----
-brew install node@10.15.3
+brew install node@12.14.1
```
Comment (chris529) [inline modules/ROOT/pages/setup/source.adoc:133]: 12.14.1

**Pull request merged**